src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -3027,7 +3042,27 @@ static void Main()
     }
 }
 ";
-            var comp = CompileAndVerify(source, expectedOutput: " (1, 2)", additionalRefs: new[] { ValueTupleRef, SystemRuntimeFacadeRef });
+
+            Action<ModuleSymbol> validator = (ModuleSymbol module) =>
+            {
+                var sourceModule = (SourceModuleSymbol)module;
+                var compilation = sourceModule.DeclaringCompilation;
+                var tree = compilation.SyntaxTrees.First();
+                var model = compilation.GetSemanticModel(tree);
+
+                var literal = tree.GetRoot().DescendantNodes().OfType<TupleExpressionSyntax>().First();
+                Assert.Equal(@"(null, (1, 2))", literal.ToString());
+                Assert.Null(model.GetTypeInfo(literal).Type);
+                Assert.Equal("(System.String, (System.Int32, System.Int32))", model.GetTypeInfo(literal).ConvertedType.ToTestDisplayString());
+                Assert.Equal(ConversionKind.ImplicitTupleLiteral, model.GetConversion(literal).Kind);
+
+                var nestedLiteral = literal.Arguments[1];
+                Assert.Equal(@"(1, 2)", nestedLiteral.ToString());
+                Assert.Null(model.GetTypeInfo(nestedLiteral).Type);
+                Assert.Null(model.GetTypeInfo(nestedLiteral).ConvertedType);
```

Comment:
@VSadov Could you comment on this? Is this what we should expect (null `Type` and `ConvertedType`)?
I couldn't find a test on tuple literal conversions with nested literals.